themes/default/content/challenge/startup-in-a-box.md

```diff
@@ -499,8 +855,8 @@ expect(await page.title()).toBe("Pulumi Challenge");
 await browser.close();
 ```
 
-Congratulations! You completed the first Pulumi Challenge. If you’d like to tear down all of these resources, run `pulumi destroy`. Otherwise, enjoy the new website!
+Congratulations! You completed the first Pulumi Challenge. If you'd like to tear down all of these resources, run `pulumi destroy`. Otherwise, enjoy the new website!
```

Comment:
One thing I'm also seeing is that `pulumi destroy` fails:

```
Diagnostics:
  gcp:projects:Service (gcp-api-compute.googleapis.com):
    error: deleting urn:pulumi:dev::sib-challenge::gcp:projects/service:Service::gcp-api-compute.googleapis.com: 1 error occurred:
        * Error when reading or editing Project Service pulumi-development/compute.googleapis.com: Error disabling service "compute.googleapis.com" for project "pulumi-development": Error waiting for api to disable: Error code 9, message: [Error in service 'compute.googleapis.com': Could not turn off service, as it still has resources in use.
```

Is this one of those situations in which we're enabling a service in the code, but should arguably be enabling it manually ahead of time?

Reply:
Ah, yes, that's exactly what's going on. Can we add a "preserve on delete" or similar?

Reply:
Adding `disableOnDestroy` fixes the issue; `pulumi destroy` runs cleanly now. I've added that to all applicable code listings.